controller/network/network.go

```diff
@@ -683,9 +688,9 @@ func (network *Network) AcceptMetrics(metrics *metrics_pb.MetricsMessage) {
 		metricId := "link." + link.Id.Token + ".latency"
 		if latency, ok := metrics.Histograms[metricId]; ok {
 			if link.Src.Id == router.Id {
```

Comment:
If we uses nanos in path calculation then the latency will overwhelm the other costs. Millis seems like a reasonable scale.

Reply:
Why not use fractional millis and keep the resolution, but have the magnitude you want?

Reply:
I'm betting that we're going to need to iterate on the scaling and combination of costs again. The current model seems a little tricky.

Reply:
I'll give the fractional millis a whirl. We may end up making the scaling factor configurable at some point, but I want to see how millis work for now.

Reply:
Ok... But like I said, we're probably going to need to revisit all of this again at some point. I suspect I'll want to spend some time on this stuff when I'm back in the fabric for Smart Routing 2.